lib/concurrency.sh

```diff
@@ -1,35 +1,13 @@
-calculate_concurrency() {
-  MEMORY_AVAILABLE=${MEMORY_AVAILABLE-$(detect_memory 512)}
-  WEB_MEMORY=${WEB_MEMORY-512}
-  WEB_CONCURRENCY=${WEB_CONCURRENCY-$((MEMORY_AVAILABLE/WEB_MEMORY))}
-  if (( WEB_CONCURRENCY < 1 )); then
-    WEB_CONCURRENCY=1
-  elif (( WEB_CONCURRENCY > 32 )); then
-    WEB_CONCURRENCY=32
+log_resources() {
+  if [[ $ELASTICIO_MEMORY_AVAILABLE ]]; then
+    echo "Detected $ELASTICIO_MEMORY_AVAILABLE MB available memory"
+  fi
+  if [[ $ELASTICIO_CPU_AVAILABLE ]]; then
+    echo "Detected $ELASTICIO_CPU_AVAILABLE CPU available"
   fi
-  WEB_CONCURRENCY=$WEB_CONCURRENCY
-}
-
-log_concurrency() {
-  echo "Detected $MEMORY_AVAILABLE MB available memory, $WEB_MEMORY MB limit per process (WEB_MEMORY)"
-  echo "Recommending WEB_CONCURRENCY=$WEB_CONCURRENCY"
-}
-
-detect_memory() {
-  local default=$1
-  local limit=$(ulimit -u)
-
-  case $limit in
-    256) echo "512";;
-    512) echo "1024";;
-    32768) echo "6144";;
-    *) echo "$default";;
-  esac
 }
 
-calculate_concurrency
-log_concurrency
+log_resources
 
-export MEMORY_AVAILABLE=$MEMORY_AVAILABLE
-export WEB_MEMORY=$WEB_MEMORY
-export WEB_CONCURRENCY=$WEB_CONCURRENCY
+export ELASTICIO_MEMORY_AVAILABLE=$ELASTICIO_MEMORY_AVAILABLE
```

Comment:
I guess, it's not necessary. Previously these vars were set by `calculate_concurrency`, but now you just have them in environment.